site/en/blog/fledge-api/index.md

```diff
@@ -1195,9 +1197,8 @@ The TURTLEDOVE effort is about offering a new API to address this use case while
 FLEDGE grew out of TURTLEDOVE and a collection of related proposals for modifications to better served the developers who would be using the API:
 
 -   In [SPARROW](https://github.com/WICG/sparrow):
-   [Criteo](https://www.admonsters.com/what-is-sparrow/) proposed the addition of a trusted-server
-   ("Gatekeeper") model.  FLEDGE includes a more limited use of trusted servers, for real-time data
-   lookup and aggregated reporting.
+   [Criteo](https://www.admonsters.com/what-is-sparrow/) proposed the addition of a
+   ("Gatekeeper") service model running in a [trusted execution environment](https://github.com/privacysandbox/fledge-docs/blob/main/trusted_services_overview.md#trusted-execution-environment).  FLEDGE includes a more limited use of TEEs, for real-time data lookup and aggregated reporting.
```

Comment:
Is 'trusted execution environment' lowercase in contexts like this? I know it's lowercase in the article linked to. (I'm not sure if it's a bit like referring to something like an API name.)

@heyawhite FYI